qa/tasks/cephfs/test_misc.py

```diff
@@ -636,3 +636,184 @@ def test_alloc_cinode_assert(self):
 
         ls_out = set(self.mount_a.ls("test_alloc_ino/"))
         self.assertEqual(ls_out, set({"dir1", "dir2"}))
+
+
+class TestNewFSCreation(CephFSTestCase):
+    MDSS_REQUIRED = 1
+    TEST_FS = "test_fs"
+    TEST_FS1 = "test_fs1"
+
+    def test_fs_creation_valid_ops(self):
+        """
+        Test setting fs ops with CLI command `ceph fs new`.
+        """
+        fs_ops = [["max_mds", "3"], ["refuse_client_session", "true"],
+                  ["allow_new_snaps", "true", "max_file_size", "65536"],
+                  ["session_timeout", "234", "session_autoclose",
+                   "100", "max_xattr_size", "150"]]
```

Comment:
add test for empty `set`.

Reply:
already added in `test_fs_creation_incomplete_args`